### PR TITLE
Add dev-lang/python2 build manifest

### PR DESCRIPTION
**Pull request by Pedrochu**
The output is really really long, but here's the last part of it (don't hesitate to say it if the whole thing is needed):
```
[+]              ./usr/bin/python2-config -> python2.7-config
[+]              ./usr/bin/python-config -> ./usr/bin/python2-config -> python2.7-config
[+]              ./usr/bin/2to3-2
[+]              ./usr/bin/pydoc-2
[+]              ./usr/bin/idle-2
[+]              ./usr/bin/python2.7
[+]              ./usr/bin/python -> ./usr/bin/python2 -> python2.7
[+]              ./usr/bin/python2 -> python2.7
[+]              ./usr/bin/smtpd.py
[+]              ./usr/bin/python2.7-config
[+]          (That's 4590 files.)
[+]          Creating data.tar.gz
[+]          Creating manifest.toml
[+]          Creating python2-2.7.17.nest
[+]      Wrapping dev-lang/python2-doc#2.7.17
[!]      The package is empty -- Skipping
[+]  Done!

```

### CODE REVIEW

**Comment by Arignir** [comment]
Yeah i'd like the whole content (pastebin it if it's too big), including the "those files do not belong to any package" part (if any).

File: dev-lang/python2.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            ],
+        },
+    ],
+    build_dependencies=[
+        'sys-libs/zlib-dev'
+    ]
+)
+def build(build):
+    packages = autotools.build(
+        configure=lambda: configure("--with-system-expat    \
```
Comment (Arignir) [inline dev-lang/python2.py]: configure takes a variadic number of arguments, you can simply do 
```
configure(
    '--with-XXX',
    '--with-YYY',
)
```

File: dev-lang/python2.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            'semver': '2.7.17',
+            'fetch': [{
+                'url': 'https://www.python.org/ftp/python/2.7.17/Python-2.7.17.tar.xz',
+                'sha256': '4d43f033cdbd0aa7b7023c81b0e986fd11e653b5248dac9144d508f11812ba41',
+            },
+            ],
+        },
+    ],
+    build_dependencies=[
+        'sys-libs/zlib-dev'
```
Comment (Arignir) [inline dev-lang/python2.py]: That looks like a really short list of dependencies compared to python3, are you sure you have enabled the maximum amount of options? 

For python3, therer's somewhere a recap of all enabled and disabled options at the end of the configure script. If it is also present for python2, it should help you add new build dependencies to make the python installation more generic and solid.

File: dev-lang/python2.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+            --with-system-ffi                               \
+            --without-ensurepip                             \
+            --enable-unicode=ucs4                           \
+            --enable-optimizations                          \
+        "),
+        split=drain_all_with_doc,
+    )
+
+    # After installing this package, it will remove pip3. This command is to reset pip and pip3.
+    stdlib.cmd("python3 -m pip install --force pip")
```
Comment (Arignir) [inline dev-lang/python2.py]: Wtf? Why?

(`python2` should not include `pip2`. If you want `pip2`, please make another package to be consistent with `python3`)

File: dev-lang/python2.py [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+    with stdlib.pushd(packages['dev-lang/python2'].wrap_cache):
+        # Fixes permissions for libraries to be consistent with other libraries
+        os.chmod(f'usr/lib64/libpython{build.major}.{build.minor}.so.1.0', 0o0755)
```
Comment (Arignir) [inline dev-lang/python2.py]:
```suggestion
        os.chmod(f'usr/lib64/libpython{build.major}.{build.minor}.so.*.*', 0o0755)
```

File: dev-lang/python2.py [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+        # Renaming to avoid conflicts with the binaries installed by Python3
+        os.rename('usr/bin/2to3', 'usr/bin/2to3-2')
+        os.rename('usr/bin/idle', 'usr/bin/idle-2')
```
Comment (Arignir) [inline dev-lang/python2.py]:
```suggestion
        os.rename('usr/bin/idle', 'usr/bin/idle2')
```

File: dev-lang/python2.py [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+        os.rename('usr/bin/pydoc', 'usr/bin/pydoc-2')
```
Comment (Arignir) [inline dev-lang/python2.py]:
```suggestion
        os.rename('usr/bin/pydoc', 'usr/bin/pydoc2')
```

**Review by Arignir — CHANGES_REQUESTED** [review]

File: dev-lang/python2.py (hunk 3 above)
Comment (Pedrochu) [reply dev-lang/python2.py]: makes sense

**Comment by Pedrochu** [comment]
There are no files that did not belong to any package, here's the full output
https://pastebin.com/KCVrAT3C

**Review by Arignir — APPROVED** [review]

**Pull request merged**